Documentation/ApiOverview/FlexForms/Index.rst

```diff
@@ -395,8 +395,63 @@ The key `flexform` is followed by the field which holds the Flexform data (`pi_f
 How to Access FlexForms From Fluid
 ----------------------------------
 
-Flexform settings can be read from within a Fluid template using
-:html:`{settings}`.
+If you are using an Extbase controller, FlexForm settings can be read from within a Fluid template using
+:html:`{settings}`. See the note on naming restrictions in :ref:`How to Read Flexforms From an Extbase Controller Action <_read-flexforms-extbase>`.
+
+If you defined your :typoscript:`FLUIDTEMPLATE` in TypoScript, you can assign single variables like that:
+
+.. code-block:: typoscript
+
+   my_content = FLUIDTEMPLATE
+   my_content {
+     variables {
+       categories = TEXT
+       categories.data = flexform: pi_flexform:categories
+     } 
+   }
+
+In order to have all FlexForm settings available, you can add a custom DataProcessor:
+
+.. code-block:: typoscript
+
+   my_content = FLUIDTEMPLATE
+   my_content {
+     dataProcessing {
+       10 = Your\Ext\DataProcessing\FlexFormProcessor
+     }
+   }
+
+.. code-block:: php
+   namespace Your\Ext\DataProcessing;
+   
+   use TYPO3\CMS\Core\Service\FlexFormService;
+   use TYPO3\CMS\Core\Utility\GeneralUtility;
+   use TYPO3\CMS\Frontend\ContentObject\DataProcessorInterface;
+   
+   class FlexFormProcessor implements DataProcessorInterface
+   {
+       public function process(
+           ContentObjectRenderer $cObj,
+           array $contentObjectConfiguration,
+           array $processorConfiguration,
+           array $processedData
+       ): array {
+           $flexFormService = GeneralUtility::makeInstance(FlexFormService::class); // or use Dependency Injection
```

Comment:
Since this targets master/v11 we should only show the DI version. If this is backported, `GeneralUtility::makeInstance()` can be used for v9.